exercises/practice/armstrong-numbers/test_armstrong-numbers.R

```diff
@@ -0,0 +1,50 @@
+source("./armstrong-numbers.R")
+library(testthat)
+
+context("armstrong-numbers")
+
+test_that("Zero is an Armstrong number", {
+  expect_equal(is_armstrong_number(0), TRUE)
+})
+
+test_that("Single-digit numbers are Armstrong numbers", {
+  expect_equal(is_armstrong_number(5), TRUE)
+})
+
+test_that("There are no two-digit Armstrong numbers", {
+  expect_equal(is_armstrong_number(10), FALSE)
+})
+
+test_that("Three-digit number that is an Armstrong number", {
+  expect_equal(is_armstrong_number(153), TRUE)
+})
+
+test_that("Three-digit number that is not an Armstrong number", {
+  expect_equal(is_armstrong_number(100), FALSE)
+})
+
+test_that("Four-digit number that is an Armstrong number", {
+  expect_equal(is_armstrong_number(9474), TRUE)
+})
+
+test_that("Four-digit number that is not an Armstrong number", {
+  expect_equal(is_armstrong_number(9475), FALSE)
+})
+
+test_that("Seven-digit number that is an Armstrong number", {
+  expect_equal(is_armstrong_number(9926315), TRUE)
+})
+
+test_that("Seven-digit number that is not an Armstrong number", {
+  expect_equal(is_armstrong_number(9926314), FALSE)
+})
+
+# test_that("Armstrong number containing seven zeroes", {
+#   expect_equal(is_armstrong_number(186709961001538790100634132976990), TRUE)
+# })
+# 
+# test_that("The largest and last Armstrong number", {
+#   expect_equal(is_armstrong_number(115132219018763992565095597973971522401), TRUE)
+# })
```

Comment:
Why are these tests commented?

Reply:
These fail in R (and many other languages) because of integer overflow. I think they need to be deleted but I wanted to give an R expert a chance to disagree.

Deleting also gets rid of the linter error.

Reply:
Okay makes sense. If they _are_ excluded, remember to add `include = false` to the corresponding entries in the `tests.toml` file: https://github.com/exercism/r/pull/243/files#diff-28bb739e7bc3f68b1a554f3abf5553f2a3423fe5f9d0951b68b7130b1b11cb40